docs/content/onedrive.md

```diff
@@ -402,13 +402,23 @@ It is a
 [known](https://github.com/OneDrive/onedrive-api-docs/issues/935#issuecomment-441741631)
 issue that Sharepoint (not OneDrive or OneDrive for Business) silently modifies
 uploaded files, mainly Office files (.docx, .xlsx, etc.), causing file size and
-hash checks to fail. To use rclone with such affected files on Sharepoint, you
+hash checks to fail. This will only happen for some files. To use rclone with such
+affected files on Sharepoint, you
 may disable these checks with the following command line arguments:
 
 ```
 --ignore-checksum --ignore-size
 ```
 
+Alternatively, if you have write access to the OneDrive files, it is possible to get
+OneDrive to make this problem disappear for certain files.
+Open the web interface for [OneDrive](https://onedrive.live.com) and find the 
```

Comment:
This is the link for OneDrive for Personal. Typo?

Reply:
No. I get this problem in regular OneDrive for consumers. I don't have a business or sharepoint account AFAIK (I do have an Office 365 subscription). I'll clarify the issue and add a link.

Reply:
amended the change but left the link to ondedrive